### PR TITLE
feat(ngUpgrade): support for content project from ng1->ng2

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: modules/upgrade/src/ng2_facade.ts [hunk 1]
```diff
@@ -21,23 +24,28 @@ export class Ng2ComponentFacade {
...
               private scope: angular.IScope, private parentInjector: Injector,
               private parse: angular.IParseService, private viewManager: AppViewManager,
               private protoView: ProtoViewRef) {
+    (<any>this.element[0]).id = id;
     this.componentScope = scope.$new();
+    this.childNodes = <Node[]><any>element.contents();
   }
 
-  bootstrapNg2() {
+  bootstrapNg2(): Node {
```
Comment (tbosch) [inline modules/upgrade/src/ng2_facade.ts]: Store internally.

File: modules/upgrade/src/ng_transclude.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import {Directive} from 'angular2/angular2';
+
+@Directive({selector: 'ng-transclude'})
+export class NgTransclude {
```
Comment (tbosch) [inline modules/upgrade/src/ng_transclude.ts]: Remove

File: modules/upgrade/upgrade.ts [hunk 3]
```diff
@@ -1 +1,2 @@
 export {createUpgradeModule, UpgradeModule} from './src/upgrade_module';
+export {NgTransclude} from './src/ng_transclude';
```
Comment (tbosch) [inline modules/upgrade/upgrade.ts]: Remove

File: modules/upgrade/src/ng2_facade.ts (hunk 1 above)
Comment (mhevery) [inline modules/upgrade/src/ng2_facade.ts]: done

File: modules/upgrade/src/ng_transclude.ts (hunk 2 above)
Comment (mhevery) [inline modules/upgrade/src/ng_transclude.ts]: done

File: modules/upgrade/upgrade.ts (hunk 3 above)
Comment (mhevery) [inline modules/upgrade/upgrade.ts]: done

**Pull request closed without merging**